src/current/v23.2/alter-database.md

```diff
@@ -834,6 +836,120 @@ The `lease_preferences` field is now updated to include `us-west1`.
 
 To remove the changes made in this example, [reset the Zone Config Extensions](#reset-a-regions-zone-config-extensions).
 
+#### Implement super regions
+
+In this example, we use [Zone Config Extensions]({% link {{ page.version.version }}/zone-config-extensions.md %}) to provide an alternative implementation of [Super Regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions), which are useful for [data domiciling]({% link {{ page.version.version }}/data-domiciling.md %}).
+
+For this example, you need to start a [`cockroach demo` cluster]({% link {{ page.version.version }}/cockroach-demo.md %}) using the following command, which [defines the available cluster localities using the `--demo-locality` flag]({% link {{ page.version.version }}/cockroach-demo.md %}#demo-locality):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --nodes=9 --demo-locality=region=us-east1:region=us-east1:region=us-central1:region=us-west1:region=europe-west1:region=europe-west1:region=europe-central1:region=europe-east1:region=europe-east1
+~~~
+
+Next, enter the statements below at the SQL prompt. These statements will:
+
+- Make the [`movr` database]({% link {{ page.version.version }}/movr.md %}) into a [multiregion database]({% link {{ page.version.version }}/multiregion-overview.md %}) by [setting the primary region]({% link {{ page.version.version }}/alter-database.md %}#set-primary-region).
+- Set the `movr` database to [survive region failures]({% link {{ page.version.version }}/multiregion-survival-goals.md %}#survive-region-failures).
+- Add the remaining regions we defined at cluster start time to the multi-region `movr` database (for a total of 6).
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr SET PRIMARY REGION "us-east1";
+ALTER DATABASE movr SURVIVE REGION FAILURE;
+ALTER DATABASE movr ADD REGION "us-central1";
+ALTER DATABASE movr ADD REGION "us-west1";
+ALTER DATABASE movr ADD REGION "europe-west1";
+ALTER DATABASE movr ADD REGION "europe-central1";
+ALTER DATABASE movr ADD REGION "europe-east1";
+~~~
+
+Now that our multiregion cluster is configured, we will use the `ALTER LOCALITY` statements shown below to [configure replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}) such that:
+
+- [Replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) in the `us-east1` region are only stored on nodes in the US.
+- Replicas in the `europe-central1` region are only stored on nodes in the EU.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr ALTER LOCALITY REGIONAL IN "us-east1" CONFIGURE ZONE USING
+    num_replicas = 5,
```

Comment:
We have 6 regions for the cluster, so with goal `REGION_FAILURE`, the num_replicas should be at least `numReplicas = maxFailuresBeforeUnavailability(numVotersForRegionSurvival) + (numRegions - 1)`, which is 2 + 6 - 1 = 7 in our case. I think we can do 
```
ALTER DATABASE movr ALTER LOCALITY REGIONAL IN "us-east1" CONFIGURE ZONE USING
    num_replicas = 7,
    constraints = '{+region=us-east1: 3,
                    +region=us-central1: 3,
                    +region=us-west1: 1}',
    lease_preferences = '[[+region=us-east1], [+region=us-central1]]';
```

Reply:
Thanks Jane, this is updated in the latest commit.  

I updated both the "us-east1" locality and the "europe-central1" locality with the `num_replicas=7` and the constraints to be `3,3,1`